### PR TITLE
Add support for vararg input

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

**Comment by regadas** [comment]
@ravwojdyla @nevillelyh PTAL

File: zoltar-tensorflow/src/test/java/com/spotify/zoltar/tf/TensorFlowModelTest.java [hunk 1]
```diff
@@ -86,10 +86,11 @@ public void testLoad() throws Exception {
         IrisFeaturesSpec.irisFeaturesSpec(),
         settings,
         JFeatureSpec::extractWithSettingsExample);
+    JFeatureSpec.wrap(IrisFeaturesSpec.irisFeaturesSpec()).extractWithSettingsExample(settings);
```
Comment (ravwojdyla) [inline zoltar-tensorflow/src/test/java/com/spotify/zoltar/tf/TensorFlowModelTest.java]: ?

File: zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java [hunk 2]
```diff
@@ -81,8 +81,8 @@
       final ModelT model,
       final FeatureExtractor<InputT, VectorT> featureExtractor,
       final AsyncPredictFn<ModelT, InputT, VectorT, ValueT> predictFn) {
-    return (input, timeout, scheduler) -> {
-      final List<Vector<InputT, VectorT>> vectors = featureExtractor.extract(input);
+    return (scheduler, timeout, inputs) -> {
```
Comment (ravwojdyla) [inline zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java]: What's the reason for change of order?

File: zoltar-core/src/main/java/com/spotify/zoltar/FeatureExtractFns.java [hunk 3]
```diff
@@ -45,7 +48,33 @@
     /**
      * Functional interface. Perform feature extraction.
      */
-    List<ValueT> apply(List<InputT> inputs) throws Exception;
+    List<ValueT> apply(InputT... inputs) throws Exception;
+  }
+
+  /**
+   * Generic feature extraction function, takes single raw input and should return extracted
```
Comment (ravwojdyla) [inline zoltar-core/src/main/java/com/spotify/zoltar/FeatureExtractFns.java]: `a single raw ...`

File: zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java (hunk 2 above)
Comment (regadas) [reply zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java]: varargs work best if they are the last arg in a method call

File: zoltar-tensorflow/src/test/java/com/spotify/zoltar/tf/TensorFlowModelTest.java (hunk 1 above)
Comment (regadas) [reply zoltar-tensorflow/src/test/java/com/spotify/zoltar/tf/TensorFlowModelTest.java]: oops

File: zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java (hunk 2 above)
Comment (ravwojdyla) [reply zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java]: ah - duh - that's for varargs, i see now

**Comment by ravwojdyla** [comment]
lgtm

**Pull request merged**